prow/pjutil/pjutil_test.go

```diff
@@ -701,6 +701,7 @@ func TestJobURL(t *testing.T) {
 		plank    config.Plank
 		pj       prowapi.ProwJob
 		expected string
+		wantErr  bool
```

Comment:
At least for places where you add it, please do not use a boolean for expected errors, check if the error you get is the error you expect. I have found a decent set of tests in this codebase that used the boolean approach and happened to pass because something else errored :)

(I do realize there is currently only one possible error, but that might be different in say six or 12 months)

Reply:
Would you rather compare the errors by string or define a separate error type and compare via `==` ?

Reply:
Done

Reply:
I don't really care much but if you go the string route it might be simpler to do define it on the test as string, because then you do not have to do the
```
var expectedErrStr string
if actualErr != nil {
  actualErrStr = actualErr.Error()
}
```

Reply:
Fixed